packages/code-studio/src/settings/FormattingSectionContent.tsx

```diff
@@ -614,16 +614,16 @@ export class FormattingSectionContent extends PureComponent<
             </div>
           </div>
 
-          <div className="form-row mb-3">
+          <div className="form-row mb-3" id="show_extra_group_column_div">
             <label
               className="col-form-label col-3"
               htmlFor="default-integer-format-input"
             >
               Rollup
             </label>
-            <div className="col pr-0 pt-2">
+            <div className="col pr-0 pt-2" id="show_extra_grou_column_cb">
```

Comment:
This is still incorrect. The label should have `htmlFor="default-show-group-column"` and then the Checkbox with a matching id.

We're actually misusing this in other places though. Labels are supposed to be 1 to 1 for form elements. In some previous sections we now have multiple inputs, but are using the label for the whole section. This breaks accessibility in those cases because the label corresponds to a specific input, but we actually have multiple inputs in that section. Some without a label.

Reply:
So the `Checkbox` component already sets a label. It's actually the use of `label` for String (which also has the wrong ID) that is incorrect because there's no input that label is associated with. The others like Integer are technically fine because they associate with the actual input element and then we have a checkbox that has its own label via the component